Project/src/main/java/app/FileUploadController.java

```diff
@@ -45,7 +49,7 @@ public String uploadFile(@RequestParam("file") MultipartFile file) throws IOExce
         newFile.setFileType(file.getContentType());
         newFile.setData(file.getBytes());
         newFile.setUser(super.getUser());
-
+        newFile.setStatus(ArticleStatus.SUBMITTED.toString());
```

Comment:
Is there a reason the article status has to be a string? Could we keep it as an enum

Reply:
It has to be displayed